src/gsy_e/models/strategy/state/heat_pump_state.py

```diff
@@ -143,14 +152,16 @@ def delete_past_state_values(self, current_time_slot: DateTime):
                                 self._last_time_slot(current_time_slot))
 
     def get_results_dict(self, current_time_slot: DateTime) -> Dict:
-        return {
+        retval = {
             "storage_temp_C": self.get_storage_temp_C(current_time_slot),
             "temp_decrease_K": self.get_temp_decrease_K(current_time_slot),
             "temp_increase_K": self.get_temp_increase_K(current_time_slot),
             "energy_consumption_kWh": self.get_energy_consumption_kWh(current_time_slot),
             "max_energy_demand_kWh": self.get_max_energy_demand_kWh(current_time_slot),
             "min_energy_demand_kWh": self.get_min_energy_demand_kWh(current_time_slot),
+            "total_traded_energy_kWh": self._total_traded_energy_kWh[current_time_slot],
```

Comment:
minor, but it would be consistent to also add a getter method for this

Reply:
As discussed offline, the total_traded_energy is not used outside of the scope of this class, therefore a potential `get_total_traded_energy_kWh` would not be used by another part of the code. We can add it once we need it.